src/packagedcode/win_pe.py

```diff
@@ -311,13 +311,13 @@ def parse(cls, location, package_only=False):
 
         License = get_first(infos, 'License')
 
-        extracted_license_statement = {}
-        if LegalCopyright or LegalTrademarks or License:
-            extracted_license_statement = dict(
-                LegalCopyright=copyr,
-                LegalTrademarks=LegalTrademarks,
-                License=License
-            )
+        extracted_license_statement = None
```

Comment:
Good catch, but you still want to check for LegalCopyright are it may contain licenses.
See for instance:
- https://github.com/blackjack4494/youtube-dlc/blob/175120c55dcd1d86ea13f21314bf175ce460cb51/win/ver.txt#L38
- https://github.com/matro7sh/myph/blob/8de7c2b811f38b18caec5f608385893f82bc47ff/README.md?plain=1#L153
- https://github.com/topcss/docker-pull-tar/blob/05cba8226750a359536351d34aee68d6ae157aa4/version.txt#L23